tools/ci.sh

```diff
@@ -4,8 +4,11 @@ export RUST_BACKTRACE=1
 export CARGO_INCREMENTAL=0
 export CARGO_REGISTRIES_CRATES_IO_PROTOCOL=sparse
 
-cargo build -p lalrpop
-cargo test --all --all-features
+cargo build --bin lalrpop --features pico-args
+# build with minimal amount of features to make sure dependencies can build too.
```

Comment:
This comment doesn't exactly match the command, as feature-powerset tests several combinations of features, not only the minimal amount of features.
Btw, does it even work as expected to first build multiple combinations and then test multiple combinations?

Reply:
you cane see the various combinasion of features for build and tests over here: https://github.com/lalrpop/lalrpop/actions/runs/4843403960/jobs/8630997943#step:5:142

Reply:
Yes, that's exactly what I expected. But it's not "the minimal amount of features" - it's basically every possible combination of features.
But of course that's only a minor nit pick.
And while I'm nit-picking: What additional value does the `cargo hack build` provide over the later `cargo hack test`? Doesn't `cargo test` implicitly build the package first?

Reply:
Ah right, when it was introduced, there was extra `[dev-dependencies]` to enable specific regex-syntax features during tests. So build & test weren't the same.
They should be the same now indeed.
Oh and initially it was using `--each-feature` rather than `--feature-powerset` hence the "minimal amount".
But yeah, this comment is a bit off now.